word_occupancy/src/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <sys/stat.h>
+#include "print.h"
+#include "make_word_table.h"
+
+int main(int argc, char** argv)
+{
+    std::unordered_map<std::string, int> umap;
+    std::string file_name = argv[1];
```

Comment:
What if no argument was passed? it throws Abort. fix it

Reply:
+